pyanaconda/bootloader.py

```diff
@@ -1677,6 +1677,119 @@ def check(self):
 
         return ret
 
+class EFIGRUB1(GRUB):
+    packages = ["efibootmgr"]
+    can_dual_boot = False
+    _config_dir = "efi/EFI/redhat"
```

Comment:
This should be modified to use the efi_dir like EFIGRUB does. That way it gets set to fedora or rhel based on the installclass.